pkg/netstate/client_test.go

```diff
@@ -21,6 +22,7 @@ import (
 
 func TestNetStateClient(t *testing.T) {
 	logger, _ := zap.NewDevelopment()
+	fmt.Println("in clinet: ", os.Getenv("API_KEY"))
```

Comment:
I think this has to use the logger, like `logger.Debug("in client: ", os.Getenv("API_KEY"))`

Reply:
yeah; that was suppose to not be there; might be a side effect from the git rebase